nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -33,12 +33,12 @@ let
   occ = pkgs.writeScriptBin "nextcloud-occ" ''
     #! ${pkgs.runtimeShell}
     cd ${cfg.package}
-    sudo=exec
+    su=exec
     if [[ "$USER" != nextcloud ]]; then
-      sudo='exec /run/wrappers/bin/sudo -u nextcloud --preserve-env=NEXTCLOUD_CONFIG_DIR --preserve-env=OC_PASS'
```

Comment:
This probably needs a different approach: su doesn't take --preserve-env, so this would need to be --preserve-environment but you might not want the _entire_ environment.

Reply:
I have changed this, and currently su will perserve the env.
Seeing as this is being run by systemd services, etc. most of the time it should be OK.

I don't think there should be any issues with doing this.
I will check later/tomorrow and see how it works.

Are there any objections to this?